packages/create-sitecore-jss/src/InitializerFactory.ts

```diff
@@ -8,6 +12,8 @@ export class InitializerFactory {
         return new NextjsInitializer();
       case 'nextjs-styleguide':
         return new NextjsStyleguideInitializer();
+      case 'node-headless-ssr-proxy':
+        return new NodeHeadlessSsrProxyInitializer();
```

Comment:
Question not related to this PR. I see that we hardcode all initializers in the factory, during initializers flow refactoring, did we expect that we will not have hardcoded values here and we will get everything automatically?

Reply:
I don't think we discussed it, but we probably could make this dynamic now.

Reply:
Yeah, I think we planned on circling back on this at some point. Worth exploring for sure. I'll add a ticket.